README.md

```diff
@@ -1,38 +1,96 @@
-# Clipper Prediction Server
+# Clipper
 
-[![License](http://img.shields.io/:license-Apache%202-red.svg)](LICENSE)
+## Project Overview
 
-## About the Project
+![Clipper System Overview](images/clipper_arch.png)
 
-![Clipper System Overview](images/arch_diagram.png)
+## What is Clipper?
 
+Clipper is a prediction serving system that sits between user-facing applications and a wide range of commonly used machine learning frameworks.
 
-Machine learning is being deployed in a growing number of applications which demand real-time, accurate, and robust predictions under heavy query load. However, most machine learning frameworks and systems only address model training and not deployment.
+## What does Clipper do?
 
-Clipper is the first general-purpose low-latency prediction serving system.  Interposing between end-user applications and a wide range of machine learning frameworks, Clipper introduces a modular architecture to simplify model deployment across frameworks.  Furthermore, by introducing caching, batching, and adaptive model selection techniques, Clipper reduces prediction latency and improves prediction throughput, accuracy, and robustness without modifying the underlying machine learning frameworks.
+* Clipper **simplifies integration of machine learning techniques** into user facing applications by providing a simple standard prediction and feedback REST interface across a wide range of commonly used machine learning frameworks.  *Clipper makes product teams happy.*
```

Comment:
by providing a simple, standard REST interface for prediction and feedback across...